base.go

```diff
@@ -55,6 +55,20 @@ func (b *Base) SetConfig(config *Config) {
 	b.config = config
 }
 
+// SetErrorChan will register a channel as the consumer of internal error
+// events.  This channel will be closed once ShutdownLoggers has finished.
```

Comment:
I think closing this should be up to whatever created it otherwise this leads to an inconsistent ownership contract.  Is there any reason ownership is transferred to gomol instead of just allowing the user to close it when they're done with it?